pom.xml

```diff
@@ -21,7 +21,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.apache.jclouds.examples</groupId>
   <artifactId>jclouds-examples</artifactId>
-  <version>2.1.0</version>
+  <version>2.2.0-SNAPSHOT</version>
```

Comment:
Even if dimension data (in labs) uses the snapshot, keep this version as-is, please.

Reply:
Great thanks I wasnt sure what the correct approach was for the parent pom.